loadrunner_professional/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e6b5ab52-139d-4dde-a4ad-94fedeac7f29",
+  "app_id": "loadrunner-professional",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your LoadRunner Professional metrics to DataDog.",
+    "title": "LoadRunner Professional",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog Configuration Window",
+        "image_url": "images/datadog_configuration_window.png"
+      }
```

Comment:
This can be removed unless you want this image to show up in the media carousel on your tile.

Instead, please add a few images of your Datadog dashboard, and/or any relevant screenshots of your platform that you'd like to display on your tile.

Reply:
i set the "banner_img": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/loadrunner_professional/assets/logos/loadrunner_logo_4x3.png", the logo will not be shown until the integration in released, any workaround?

Reply:
"Kubernetes Cluster Overview Dashboard" has this in the "Image URL" field of the banner: "/static/images/integration_dashboard/kubernetes_hero_2.jpeg"

Reply:
Just checked with the team - if you'd like to host your image on GitHub and display it on your dashboard, the link will be the following once the PR is merged:

https://github.com/DataDog/integrations-extras/tree/master/loadrunner_professional/<path-to-image>